examples/visualization/make_report.py

```diff
@@ -26,7 +26,7 @@
 # Do standard folder parsing (this can take a couple of minutes):
 
 report = Report(image_format='png', subjects_dir=subjects_dir,
-                info_fname=evoked_fname, subject='sample')
+                info_fname=evoked_fname, subject='sample', raw_psd=True)
```

Comment:
why is it a boolean flag? why not plot it always?

Reply:
Backward compat / not changing the output. I didn't want to force a slow-to-compute option on people, but if it seems useful enough I guess it's easy to disable so we could switch it to `True` default.